slack/backend/app/src/main/kotlin/com/ritense/slack/listener/ApplicationReadyEventListener.kt

```diff
@@ -0,0 +1,25 @@
+package com.ritense.slack.listener
+
+import com.ritense.document.domain.event.DocumentDefinitionDeployedEvent
+import com.ritense.document.service.DocumentDefinitionService
+import com.ritense.valtimo.contract.authentication.AuthoritiesConstants
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+
+@Component
+class ApplicationReadyEventListener(
+    private val documentDefinitionService: DocumentDefinitionService,
+) {
+
+    @EventListener(DocumentDefinitionDeployedEvent::class)
+    fun handleDocumentDefinitionDeployed(event: DocumentDefinitionDeployedEvent) {
+        setDocumentDefinitionRole(event)
+    }
+
+    fun setDocumentDefinitionRole(event: DocumentDefinitionDeployedEvent) {
+        documentDefinitionService.putDocumentDefinitionRoles(
+            event.documentDefinition().id().name(),
+            setOf(AuthoritiesConstants.ADMIN, AuthoritiesConstants.USER)
+        )
+    }
```

Comment:
I don't think the plugin is responsible for what is done here

Reply:
@Rick-Ritense all code in the 'app' module can be considered as not part of the plugin. The app module only exists to develop/run the plugin locally.

Reply:
The slack plugin come together with a example/test application. That is an application that has a single document, a single process and a single Slack plugin.